src/components/flyout/flyout_body.tsx

```diff
@@ -31,9 +33,21 @@ export const EuiFlyoutBody: EuiFlyoutBodyProps = ({
     'euiFlyoutBody__overflow--hasBanner': banner,
   });
 
+  const euiTheme = useEuiTheme();
+  const styles = euiFlyoutBodyStyles(euiTheme);
+
+  const cssStyles = [
+    styles.euiFlyoutBody,
+    banner && styles.euiFlyoutBody__banner,
+  ];
+  const overflowCSSStyles = [
```

Comment:
small camel casing nit
```suggestion
  const overflowCssStyles = [
```